code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -9,67 +8,48 @@
 	color = "#dcd9cd"
 	taste_description = "boiled cabbage"
 	unaffected_species = IS_MACHINE
+	var/kois_type = 1
 
 /datum/reagent/kois/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
-	if(isvaurca(M))
+	if(alien == IS_VAURCA)
 		M.heal_organ_damage(1.2 * removed, 1.2 * removed)
 		M.adjustToxLoss(-1.2 * removed)
 		M.nutrition += nutriment_factor * removed // For hunger and fatness
 		M.add_chemical_effect(CE_BLOODRESTORE, 6 * removed)
 	else
+		M.adjustToxLoss(1 * removed)
 		if(istype(M,/mob/living/carbon/human))
-
 			var/mob/living/carbon/human/H = M
-			if(!H.internal_organs_by_name["kois"])
-
-				if(prob(10*removed))
-					var/obj/item/organ/external/affected = H.get_organ("chest")
-					var/obj/item/organ/parasite/kois/infest = new()
-					infest.replaced(H, affected)
-
-				else
-					H.adjustToxLoss(1 * removed)
-		else
-			M.adjustToxLoss(1 * removed)
-		return
+			switch(kois_type)
+				if(1) //Normal
+					if(!H.internal_organs_by_name["kois"] && prob(10*removed))
```

Comment:
This probability, and the one below it should be reduced by some margin. After speaking with Fowl about this, we determined that kois infections should be "rare". Atm, this evaluates to `10 * 0.2 * 4 = 8%` every Life() tick, until you've managed to ingest all of it. This is an incredibly high chance from a statistical perspective, and could probably be slashed, if not brought down by an order of magnitude.

Reply:
Is there a specific value you desire?